extensions/amp-list/0.1/amp-list.js

```diff
@@ -317,7 +317,7 @@ export class AmpList extends AMP.BaseElement {
       const attributes = dict({
         'ampListAttributes': {
           'items': this.element.getAttribute('items') || 'items',
-          'singleItem': this.element.getAttribute('single-item'),
```

Comment:
We should clarify in the PR description why `getAttribute` won't work (e.g., it doesn't work because different browsers have inconsistent behaviors on whether to return an empty string when the attribute doesn't exist).

Reply:
Done. 
However it's working, our main concern was to keep consistent with normal amp-list code (and also make the code in viewer easier).